lib/collection/src/collection/collection_ops.rs

```diff
@@ -194,14 +194,13 @@ impl Collection {
                 });
             }
 
-            if peers.len() == 1 {
+            // Check that we are not removing the
```

Comment:
It's just weird to not finish a comment like that 😂

```suggestion
            // Check that we are not removing the last active replica
```